File: OCH_Tao/ABC/B/057.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const [N, M] = ((input.shift()).split(" ")).map(x => parseInt(x, 10));
let s = [];
for (let i = 0; i < N; i++) {
  s.push(((input.shift()).split(" ")).map(x => parseInt(x, 10)));
}
let c = [];
for (let i = 0; i < M; i++) {
  c.push(((input.shift()).split(" ")).map(x => parseInt(x, 10)));
}
for (let i = 0; i < N; i++) {
  let l = [];
  for (let j = 0; j < M; j++) {
    l.push(Math.abs(s[i][0] - c[j][0]) + Math.abs(s[i][1] - c[j][1]));
  }
  console.log(l.indexOf(Math.min(...l)) + 1);
}

File: OCH_Tao/ABC/B/057.py
N,M = map(int,input().split())
S = []
for i in range(N):
  S.append(list(map(int,input().split())))
C = []
for i in range(M):
  C.append(list(map(int,input().split())))
for i in range(N):
  l = []
  for j in range(M):
    l.append(abs(S[i][0]-C[j][0])+abs(S[i][1]-C[j][1]))
  else:
    print(l.index(min(l))+1)